### PR TITLE
Release 5.9-rc1

### PR DESCRIPTION
**Pull request by Beep6581**

### CODE REVIEW

**Comment by Thanatomanic** [comment]
Looks good! Would you please also document the steps you have taken so far, for future reference? Perhaps just elaborate things on https://rawpedia.rawtherapee.com/How_to_release_RawTherapee.

**Comment by ghost** [comment]
Just gave this 5.9 rc1 appimage a quick try and it runs smoothly on a, somewhat older, Linux (Debian 10.13/Buster) box.

**Comment by Beep6581** [comment]
@Thanatomanic I don't think I did anything not in that document. Going through the log and looking at some patches took a while. It was good to catch up, and you did a good job, e.g. when you reworked the logo. I used my old laptop with the non-"fixed" version of `sort`. Speaking of that, we might need to drop the reliance on natural-sort since the coreutils devs consider having it drop lines [the correct behavior](https://github.com/Beep6581/RawTherapee/pull/6503#issuecomment-1262042640). Next I'll try to update the screenshots in the appdata file, and then the daunting task of setting up the website toolchain on my desktop computer and documenting the process.

File: RELEASE_NOTES.txt [hunk 1]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+
+
+
+NEW FEATURES SINCE 5.8
+
+- The Spot Removal tool (Detail tab) was added, for removing dust specks and small objects.
+- The Color Appearance & Lighting tool (Advanced tab), formerly known as CIECAM02, now includes CAM16. By taking into account the conditions of the photographed scene and the conditions under which the image is viewed, it allows you to adjust the image in a way which matches human color perception.
+- The Local Adjustments tool (Local tab) was added, for performing a wide range of operations on an area of the image determined by its geometry or color.
+- The Wavelet Levels tool (Advanced tab) received various improvements.
+- The White Balance tool (Color tab) received new automatic white balance methods.
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]: Only one new method was added (temperature correlation).

File: RELEASE_NOTES.txt [hunk 2]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Nikon D5100
+    - Nikon D5100
+    - Nikon D6
+    - Nikon D70s
+    - Nikon D780
+    - Nikon D810A
+    - Nikon Z 5
+    - Nikon Z 50 (DCP)
+    - Nikon Z 6_2
+    - Nikon Z 7_2
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Nikon Z 6II
    - Nikon Z 7II
```
https://imaging.nikon.com/lineup/mirrorless/index.htm

File: RELEASE_NOTES.txt [hunk 3]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Nikon Z fc
+    - Nikon Z fc
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Nikon Z fc
```

File: RELEASE_NOTES.txt [hunk 4]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - OLYMPUS E-M10MarkIV
+    - OLYMPUS E-M1MarkIII
+    - OLYMPUS E-M1X
+    - OLYMPUS E-M5MarkII (DCP)
+    - OLYMPUS E-M5MarkIII
+    - OLYMPUS E-PL10
+    - OLYMPUS E-PL9
+    - OLYMPUS STYLUS1
+    - OLYMPUS STYLUS1,1s
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - OLYMPUS Stylus 1
    - OLYMPUS Stylus 1s
```
I think this is "Stylus 1" and "Stylus 1s"

File: RELEASE_NOTES.txt [hunk 5]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Panasonic DC-G99
+    - Panasonic DC-S1H
+    - Panasonic DC-S5 (DCP)
+    - Panasonic DC-TZ95
+    - Panasonic DC-ZS80
+    - Panasonic DMC-TZ80
+    - Panasonic DMC-TZ85
+    - Panasonic DMC-ZS60
+    - RICOH PENTAX K-1 Mark II
+    - RICOH PENTAX K-3 MARK III
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - RICOH PENTAX K-3 Mark III
```
https://us.ricoh-imaging.com/wp-content/uploads/2019/01/k-3-3.pdf

File: RELEASE_NOTES.txt [hunk 6]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+- The Haze Removal tool (Detail tab) received a saturation adjuster.
+- The RawTherapee theme was improved, including changes to make it easier to see which tools are enabled.
+- The Navigator (Editor tab) can now be resized.
+- The Resize tool (Transform tab) now allows to resize by the long or short edge.
+- The Crop tool (Transform tab) received a "centered square" crop guide, useful when the resulting non-square image will also be used on social media which crop to a square format.
+- The Pixel Shift demosaicing method (Raw tab) now allows using an average of all frames for regions with motion.
+- Added or improved support for cameras, raw formats and color profiles:
+    - Canon EOS 100D / Rebel SL1 / Kiss X7
+    - Canon EOS 1DX Mark III
+    - Canon EOS 1DX Mark III
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Canon EOS 1DX Mark III
```

File: RELEASE_NOTES.txt [hunk 7]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Canon EOS 2000D / Rebel T7 / Kiss X90
+    - Canon EOS 400D DIGITAL
+    - Canon EOS 5D Mark II
+    - Canon EOS 5D Mark II
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Canon EOS 5D Mark II
```

File: RELEASE_NOTES.txt [hunk 8]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Canon EOS 5D Mark IV (DCP)
+    - Canon EOS 90D (DCP)
+    - Canon EOS M6 Mark II (DCP)
+    - Canon EOS R (DCP)
+    - Canon EOS R3, R7 and R10
+    - Canon EOS R3, R7 and R10.
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Canon EOS R3, R7 and R10
```

File: RELEASE_NOTES.txt [hunk 9]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Canon EOS R5 (DCP)
+    - Canon EOS R6 (DCP)
+    - Canon EOS RP
+    - Canon EOS-1D Mark III
+    - Canon EOS-1Ds
+    - Canon EOS-1Ds Mark II
+    - Canon EOS-1Ds Mark II
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Canon EOS-1Ds Mark II
```

File: RELEASE_NOTES.txt [hunk 10]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - FUJIFILM X-T2
+    - FUJIFILM X-T3 (DCP)
+    - FUJIFILM X-T30
+    - FUJIFILM X-T4
+    - FUJIFILM X100V
+    - Fujifilm GFX 100
+    - Fujifilm GFX100S though lossy compression and alternative crop modes (e.g. 4:3) are not supported yet
+    - Fujifilm X-A20
+    - Fujifilm X-T4
+    - Fujifilm X-T4
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Fujifilm X-T4
```

File: RELEASE_NOTES.txt [hunk 11]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - NIKON Z 6 (DCP)
+    - NIKON Z 7 (DCP)
+    - Nikon 1 J4
+    - Nikon COOLPIX P950
+    - Nikon D2Hs
+    - Nikon D2Xs
+    - Nikon D300s
+    - Nikon D3500
+    - Nikon D5100
+    - Nikon D5100
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Nikon D5100
```

File: RELEASE_NOTES.txt [hunk 12]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Nikon Z 50 (DCP)
+    - Nikon Z 6_2
+    - Nikon Z 7_2
+    - Nikon Z fc
+    - Nikon Z fc
+    - OLYMPUS E-M10MarkIV
+    - OLYMPUS E-M1MarkIII
+    - OLYMPUS E-M1X
+    - OLYMPUS E-M5MarkII (DCP)
+    - OLYMPUS E-M5MarkIII
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - OLYMPUS E-M10 Mark IV
    - OLYMPUS E-M1 Mark III
    - OLYMPUS E-M1X
    - OLYMPUS E-M5 Mark II (DCP)
    - OLYMPUS E-M5 Mark III
```

File: RELEASE_NOTES.txt [hunk 13]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Panasonic DC-ZS80
+    - Panasonic DMC-TZ80
+    - Panasonic DMC-TZ85
+    - Panasonic DMC-ZS60
+    - RICOH PENTAX K-1 Mark II
+    - RICOH PENTAX K-3 MARK III
+    - SONY ILCE-9 (DCP)
+    - SONY NEX-7
+    - Samsung Galaxy S7
+    - Samsung Galaxy S7
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Samsung Galaxy S7
```

**Review by Lawrence37 — CHANGES_REQUESTED** [review]
@Beep6581 Spotted a few issues in the release notes.

File: RELEASE_NOTES.txt [hunk 14]
```diff
@@ -25,16 +18,182 @@ In order to use RawTherapee efficiently you should know that:
...
+    - Sony DSC-RX10M4
+    - Sony DSC-RX1R
+    - Sony ILCE-1
+    - Sony ILCE-6100
+    - Sony ILCE-6400 (DCP)
+    - Sony ILCE-6600 (DCP)
+    - Sony ILCE-7C
+    - Sony ILCE-7M3
+    - Sony ILCE-7M4
+    - Sony ILCE-7M4
```
Comment (Lawrence37) [inline RELEASE_NOTES.txt]:
```suggestion
    - Sony ILCE-7M4
```

**Review by Lawrence37 — APPROVED** [review]

File: rtdata/languages/Japanese [hunk 15]
```diff
@@ -1,15 +1,4 @@
...
-#03 2011-11-20 a3novy
-#04 2011-12-03 a3novy
-#05 2012-02-11 a3novy
-#06 2012-04-04 a3novy
-#07 2012-07-12 a3novy
-#08 2012-12-22 a3novy
-#09 2013-04-01 a3novy
-#10 2013-04-19 a3novy
-#11 2020-06-24 Yz2house
-#12 2022-06-04 Yz2house
```
Comment (Lawrence37) [inline rtdata/languages/Japanese:12]: Was this unintentionally removed?

**Pull request merged**